### PR TITLE
Add the web page for October meetup

### PR DESCRIPTION
**Pull request by YenPeiChen07**
@yungyuc I have created a page for the October meetup. And renamed `content/pages/meetup/insex.rst` with `content/pages/meetup/index.rst`. Please have a check.

For the first section about the synopsis of the event, I am not quite sure whether it is good for now or just keep the content as same as the May (https://sciwork.dev/sprint/2022/05-nycu-career). Feel free to give me some comments.

### CODE REVIEW

**Comment by yungyuc** [comment]
Thanks a lot for spotting the typo!

File: content/pages/meetup/2022/10-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
+==============================================
+Sciwork Meetup October 2022 in NYCU
+==============================================
+
+:date: 2022-09-13 18:00
+:url: meetup/2022/10-nycu
+:save_as: meetup/2022/10-nycu.html
+
+Open, Free, and Balanced Software Career: a Panel Discussion
```
Comment (yungyuc) [inline content/pages/meetup/2022/10-nycu.rst:9]: Need a different title

File: content/pages/meetup/2022/10-nycu.rst [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+| Address: EC329 No. 1001, University Road, Hsinchu 300, Taiwan
+|   地址: 新竹市東區大學路 1001 號 工程三館 329 室
+|
+
+.. raw:: html
+
+  <div style="overflow:hidden; padding-bottom:56.25%; position:relative; height:0;">
+    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3622.243194966577!2d120.99445851483667!3d24.78712468408855!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3468360f96adabd7%3A0xedfd1ba0fa6c6bf7!2z5Lqk6YCa5aSn5a245bel56iL5LiJ6aSo!5e0!3m2!1szh-TW!2stw!4v1663061149092!5m2!1szh-TW!2stw" 
+        width="600" height="450" style="border:0;" allowfullscreen=""
```
Comment (yungyuc) [inline content/pages/meetup/2022/10-nycu.rst:93]: Could you make the embedded map to be as wide as the page?

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: content/pages/meetup/2022/10-nycu.rst (hunk 1 above)
Comment (YenPeiChen07) [reply content/pages/meetup/2022/10-nycu.rst:9]: Yes... But I have not come up with a suitable tile. Do you have any ideas?

File: content/pages/meetup/2022/10-nycu.rst (hunk 2 above)
Comment (YenPeiChen07) [reply content/pages/meetup/2022/10-nycu.rst:93]: Sure, let me change it.

File: content/pages/meetup/2022/10-nycu.rst (hunk 1 above)
Comment (yungyuc) [reply content/pages/meetup/2022/10-nycu.rst:9]: What about repeating the HTML title in the same way as other sprints, e.g., https://sciwork.dev/sprint/2022/09-hsinchu ?

Comment (YenPeiChen07) [reply content/pages/meetup/2022/10-nycu.rst:9]: Ok~ That is quite clear.

**Comment by YenPeiChen07** [comment]
@yungyuc I have made changes for the title and the embedded map.

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
> @yungyuc I have made changes for the title and the embedded map.

Thanks.  LGTM.

**Pull request merged**